config/locales/en.yml

```diff
@@ -16,6 +16,7 @@ en:
     contact_method: "contact-method"
     gp: "gp"
     address: "address"
+    health_1: "health-1"
```

Comment:
Need to extend this to `health_8:`

Reply:
I'm surprised to see that these, and other translations in this section, are necessary. Is the default behaviour not to `dasherize` these? Where are they used?